packages/clerk-js/src/ui/components/PricingTable/PricingTableDefault.tsx

```diff
@@ -8,7 +8,6 @@ import * as React from 'react';
 
```

Comment:
General note, that all of the new elements we're adding in the pricing table here should have descriptors added to them.